lunatrace/bsl/hasura/metadata/databases/lunatrace/tables/vulnerability_cwe.yaml

```diff
@@ -6,34 +6,38 @@ insert_permissions:
     permission:
       check: {}
       columns:
-        - id
-        - name
+        - common_name
         - description
         - extended_description
+        - id
+        - name
 select_permissions:
   - role: service
     permission:
       columns:
-        - id
+        - common_name
```

Comment:
Did you find a need to create groups of CWEs or are the common names all distinct?